lib/ebay_request/business_policies.rb

```diff
@@ -49,8 +49,12 @@ def auth_header
   end
 
   def errors_for(response)
-    [response.dig("errorMessage", "error")].flatten.compact.map do |err|
-      [err["severity"], err["errorId"], err["message"]]
+    [response.dig("errorMessage", "error")].flatten.compact.map do |error|
+      EbayRequest::ErrorItem.new(
```

Comment:
With dry-initializer-based item you can use just:

```ruby
EbayRequest::ErrorItem.new(error)
```

Reply:
Different APIs have different error format. It is better to keep explicit mapping IMO.